service/src/main/resources/db/migration/V262__child_absences_function.sql

```diff
@@ -0,0 +1,136 @@
+CREATE FUNCTION absences_in_period(childIds uuid[], period daterange)
```

Comment:
Nit: the indentation in this file is a bit inconsistent, with quite wide indentations in some parts (like in the TABLE parenthesis near the start).

Reply:
This is what the autoformatting in IDEA does currently 🤷
The formatter could be adjusted, but I'll leave that as a separate thing.